cpp/kiss_icp/pipeline/KissICP.cpp

```diff
@@ -107,4 +107,11 @@ bool KissICP::HasMoved() {
     return motion > 5.0 * config_.min_motion_th;
 }
 
+void KissICP::Reset(Eigen::Quaterniond quaternion, Eigen::Vector3d translation) {
```

Comment:
This method does not reset the poses but instead injects a new one... I'd split this into two methods. What do you think?

Reply:
Alright, My understanding from your comment, 
1 . Reset to be done when we Reset to the origin position.
2 . Since This feature is setting the starting position, we need to have a method SetPose.
Is that what you are referring to?